typescript/packages/wallets/aptos/CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# @goat-sdk/wallet-solana
```

Comment:
No need to add this file, it will get generated automatically for the aptos package!

Reply:
thanks! removed